04_numeronym/davidbroughsmyth/main_test.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+	main()
+	expected := strconv.Quote("[a11y K8s abc]\n")
+	actual := strconv.Quote(buf.String())
+	assert.Equal(t, expected, actual, "Unexpected output from main()")
+}
+
+var tests = map[string]struct {
+	input, want []string
+}{
+	"empty":    {input: []string{}, want: []string{}},
```

Comment:
@davidbroughsmyth LGTM. My suggestion is to add a few more tests, special characters, alphanumeric strings.

Reply:
@sirigithub I've added a few more tests as suggested.

Reply:
@sirigithub I've added more test to handle the suggest test conditions